extensions/amp-ad-network-doubleclick-impl/0.1/test/test-doubleclick-a4a-config.js

```diff
@@ -99,6 +100,21 @@ describe('doubleclick-a4a-config', () => {
           DOUBLECLICK_EXPERIMENT_FEATURE.CANONICAL_EXPERIMENT);
     });
 
+    it('should select into non-SSL canonical AMP experiment when not on CDN',
+        () => {
+          forceExperimentBranch(mockWin, DFP_CANONICAL_FF_EXPERIMENT_NAME,
```

Comment:
Modify to verify that maybeSelectExperiment is called as expected instead of forcing into experiments.